src/EnergyPlus/UnitarySystem.cc

```diff
@@ -6791,55 +6791,72 @@ namespace UnitarySystems {
                 if (state.dataHVACFan->fanObjs[FanIndex]->speedControl == HVACFan::FanSystem::SpeedControlMethod::Discrete) {
                     int NumSpeeds = state.dataHVACFan->fanObjs[FanIndex]->m_numSpeeds;
                     if (NumSpeeds > 1) {
```

Comment:
Made the following very targeted to these coil types, only for WSHP, only if air flow is not autoszied for both WSHP parent and accompanied fan objects. Also, mass flow should not need to be set here but I left those lines of code, mass flow should get overwritten later unless that is somehow protected.